packages/suite/src/components/wallet/WalletLayout/AccountTopPanel/AccountNavigation.tsx

```diff
@@ -54,6 +56,16 @@ export const AccountNavigation = () => {
             activeRoutes: ['wallet-tokens', 'wallet-tokens-hidden'],
             'data-testid': '@wallet/menu/wallet-tokens',
         },
+        {
+            id: 'wallet-nfts',
+            callback: () => {
+                goToWithAnalytics('wallet-nfts', { preserveParams: true });
+            },
+            title: <Translation id="TR_NAV_NFTS" />,
+            isHidden: !['ethereum'].includes(networkType),
```

Comment:
would make sense to use `getNetworkFeatures(symbol).includes('nft-definitions');`

Reply:
yes but now there is be "nft" in features, will use that